CONTRIBUTING.md

```diff
@@ -19,8 +19,17 @@ When adding a new or updating an existing community member profile be sure to fo
 
 #### Badges for Community Members
 
-Badges are a way to identify projects community members have made contributions to. When a community member successfully completes a project, they get the badge of that community. An example of how a badge is written can be found [here](https://github.com/layer5io/layer5/blob/master/src/collections/members/abishek-kumar/index.mdx), and it will be displayed like [this](https://layer5.io/community/members/abishek-kumar) on the community project.
+Badges are a way to identify projects community members have made contributions to. These badges are assigned to a contributor to acknowledge the contributions made to the respective project. An example of how a badge is written can be found [here](https://github.com/layer5io/layer5/blob/master/src/collections/members/abishek-kumar/index.mdx), and it will be displayed like [this](https://layer5.io/community/members/abishek-kumar) on the community project.
```

Comment:
```suggestion
Badges offer recognition of the works of community members. A variety of badges exist so that community members and their efforts may be affiliated with a particular project or community initiative. Badges are assigned to a community member in acknowledgement of their engagement within and/or contribution to the representative project or (sub-)community. An example of how a badge assigned using markdown can be found [here](https://github.com/layer5io/layer5/blob/master/src/collections/members/abishek-kumar/index.mdx), and an example of how badges are shown on a member profile is [here](https://layer5.io/community/members/abishek-kumar).
```